app/lib/LevelBus.coffee

```diff
@@ -249,6 +252,21 @@ module.exports = class LevelBus extends Bus
     # don't let what the server returns overwrite changes since the save began
     tempSession = new LevelSession _id: @session.id
     tempSession.save(patch, {patch: true, type: 'PUT'})
+    
+  updateSessionConcepts: ->
+    return unless @session.get('codeLanguage') in ['javascript', 'python']
+    try
+      tags = tagger({ast: @session.lastAST, language: @session.get('codeLanguage')})
+      tags = _.without(tags, 'basic_syntax')
+      @session.save('codeConcepts', tags)
```

Comment:
I don't think we should do a separate save here, we should just let the save that will happen on line 254 just below this take care of the save.

Reply:
Whoops, good catch. Meant it to be `set`. Fixed this line.